privacy-protections/storage-partitioning/server/routes.js

```diff
@@ -93,7 +93,7 @@ router.get('/get_hsts.png', (req, res) => {
     // The X-Forwarded-Proto header is added by Glitch's proxy
```

Comment:
hm, do we have that header? Probably not, right? Because it was glitches' weirdness? If so, we need to update this code to work on our server and correctly distinguish http from https - I can look into that if you are not familiar with node. Again, if we make this change for privacy-test-pages.sites, it means that glitch.me version will no longer funciton correctly, but I think in this case it's easy to support both if needed

Reply:
I'm not too familiar with the code discrepancies that are required to correctly work between http and https here.

Reply:
Per our discussion - this flag is set correctly above (`let isHTTPS = req.protocol === 'https';`) so we should be able to safely remove the whole if block below together with the comment. Let's test -> remove -> ship it